graphics/opencv4-devel/Portfile

```diff
@@ -283,6 +284,7 @@ configure.args-append \
                     -DWITH_IMGCODEC_PFM:BOOL=ON \
                     -DWITH_QUIRC:BOOL=OFF \
                     \
+                    -DHAVE_QUIRC:BOOL=ON \
```

Comment:
You need to change this to `-DBUILD_QUIRC:BOOL=OFF` and `-DWITH_QUIRC:BOOL=ON`.

And move to the earlier section, where we pair these together.

Reply:
Are you sure about that? It’s true that I hacked in a `BUILD_QUIRC` variable into `libquirc`‘s Makefile, but that is unrelated and I’m pretty sure that this is the correct setting the the `cmake` build in this repo.

Reply:
> Are you sure about that? It’s true that I hacked in a `BUILD_QUIRC` variable into `libquirc`‘s Makefile, but that is unrelated and I’m pretty sure that this is the correct setting the the `cmake` build in this repo.

I'll double-check tomorrow.